xml/System.ServiceModel.Security/Basic128SecurityAlgorithmSuite.xml

```diff
@@ -79,7 +79,7 @@
       <Docs>
         <summary>Gets the default asymmetric signature algorithm, RsaSha1Signature.</summary>
         <value>The default asymmetric signature algorithm, RsaSha1Signature.</value>
-        <remarks>To be added.</remarks>
+        <remarks>Due to collision problems with SHA1, Microsoft recommends SHA256.</remarks>
```

Comment:
"SHA256" isn't a legal input here.  Presumably it would be something like `RsaSha256Signature`; but I don't know what type this is.  Maybe it's fine as a remark, but it seems like the extra bit of followup would be helpful)

Reply:
(This applies similarly to the other changes in this file)

Reply:
What about changing it to read 

```suggestion
        <remarks>Due to collision problems with SHA1, Microsoft recommends a security model based on SHA256 or better.</remarks>
```

This would make it not specifically talk about "this is SHA1, put in a SHA256-type value" and instead just notes that this entire type is insecure?

Reply:
@bartonjs Does this suggestion make sense?

Reply:
I like "Due to collision problems with SHA1, Microsoft recommends a security model based on SHA256 or better.", yeah.  But it should also be done to the HMAC entry (the third change in this file).

Reply:
Should we just do it for all of them?